app/models/organization.rb

```diff
@@ -253,6 +255,12 @@ def correct_logo_mime_type
     end
   end
 
+  def some_request_type_enabled
+    unless enable_child_based_requests? || enable_individual_requests || enable_quantity_based_requests
+      errors.add(:enable_requests, "You must allow at least one request type (child-based, individual, or quantity-based)")
```

Comment:
@cielf yep, I noticed that too and added this check to ensure that at least one is enabled!

Reply:
Excellent -- Thanks!